creator-node/src/apiHelpers.js

```diff
@@ -53,12 +59,54 @@ const isValidResponse = module.exports.isValidResponse = (resp) => {
 }
 
 module.exports.successResponse = (obj = {}) => {
+  // generate timestamp
+  const timestamp = new Date().toISOString()
+
+  // format data to sign
+  let toSign = { timestamp, data: { ...obj } }
+  toSign = JSON.stringify(_sortKeys(toSign))
+
+  // hash data
+  const toSignHash = web3.utils.keccak256(toSign)
+
+  // generate signature with hashed data and private key
+  const signedResponse = web3.eth.accounts.sign(toSignHash, config.get('delegatePrivateKey'))
+
   return {
     statusCode: 200,
-    object: obj
+    object: {
+      data: {
+        ...obj
+      },
+      ...obj,
```

Comment:
add a todo here to deprecate this and just mention this is for backwards compatibility